routes/api/users.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const router = express.Router();
+const { restoreUser } = require('../../auth');
+const createError = require('http-errors');
+const {asyncHandler} = require('../utils');
+const { Follow } = require("../../db/models");
+
+/* Add a follower to a user */
+router.post('/users/:id/followers', restoreUser, asyncHandler(async (req, res, next) => {
+    
+    /* Return 401 error if
+           user is not authenticated. */
+    if (!res.locals.authenticated) {
+        return next(createError(401)); 
+     }
+
+    /* Create a new follower */
+    const newFollower = await Follow.create({
+        userId: res.locals.user.id,
+        followerId: req.params.id
```

Comment:
The values for these two should be flipped:
`userId` should be `req.params.id`
`followerId` should be `res.locals.user.id`